tests/integration/test_ec2.py

```diff
@@ -1,4 +1,31 @@
+import pytest
+from botocore.exceptions import ClientError
+
 from localstack.utils.aws import aws_stack
+from localstack.utils.strings import short_uid
+
+# public amazon image used for ec2 launch temlpates
+PUBLIC_AMAZON_LINUX_IMAGE = "ami-06c39ed6b42908a36"
+PUBLIC_AMAZON_UBUNTU_IMAGE = "ami-03e08697c325f02ab"
+
+
+@pytest.fixture()
+def create_launch_template(ec2_client):
+    template_ids = []
+
+    def create(template_name):
+        response = ec2_client.create_launch_template(
+            LaunchTemplateName=template_name,
+            LaunchTemplateData={
+                "ImageId": PUBLIC_AMAZON_LINUX_IMAGE,
+            },
+        )
+        template_ids.append(response["LaunchTemplate"]["LaunchTemplateId"])
+        return response
+
+    yield create
+    for id in template_ids:
+        ec2_client.delete_launch_template(LaunchTemplateId=id)
```

Comment:
You can make sure all templates are deleted despite exceptions with:

```suggestion
        with contextlib.suppress(ClientError):
            ec2_client.delete_launch_template(LaunchTemplateId=id)
```